### DISCUSSION TITLE
Mic/StringPool-For-Your-Code

### DISCUSSION THREAD

**Discussion by giscus** [newbe36524/Newbe.Docs#259] [Announcements]
# Mic/StringPool-For-Your-Code

字符串池化，减少重复实例，内存降低，一切就是这样的轻松愉快。

https://www.newbe.pro/Mic/StringPool-For-Your-Code

**Comment by andysjt** [comment]
请问菜单里没有“Run under dotMemory Unit”是什么原因？

**Reply by newbe36524** [reply]
查看一下是不是没有单独安装 dotMemory 的套件。
